src/App.tsx

```diff
@@ -30,6 +31,7 @@ export enum Routes {
   HOME = '/',
   LOGIN = '/login',
   SIGNUP = '/signup',
+  SIGNUP_FORM = '/signup/form',
```

Comment:
What's the difference between SIGNUP and SIGNUP_FORM? Somewhat hard to tell which is which

Reply:
Ah, basically per the design we have a signup directory (you select between particpating family vs general member) which is where we want users to start, but then we have the form itself